Vokoder.podspec

```diff
@@ -56,7 +56,9 @@ Pod::Spec.new do |s|
   s.subspec 'Swift' do |sw|
     sw.ios.deployment_target = '8.0'
     sw.tvos.deployment_target = '9.0'
-    sw.dependency 'Vokoder/DataSources'
+    sw.watchos.deployment_target = '3.0'
+    
+    sw.dependency 'Vokoder/Core'
```

Comment:
The advantage of (and intention) of making this dependent on DataSources was to be able to have one line in your podfile if you wanted to use swift: ` pod 'Vokoder/Swift'`.  Changing this means you'll need two if you want to use the datasources.  If cocoapods is smart enough to combine multiple subspecs into a single framework and including multiple subspecs doesn't cause headaches, then I'm ok with this.